PterodactylLib/Built_Product/PterodactylServer.app/Contents/_CodeSignature/CodeResources

```diff
@@ -10,15 +10,15 @@
 		</data>
 		<key>Resources/Assets.car</key>
 		<data>
-		ooukSXAP480efdIWFaH2vnd9hxI=
+		8lNwsXX8BfvEkzPcsOxYb6AfhIs=
```

Comment:
this file keeps changing between PRs, isn't it? would it have a conflict with #2 ?

Reply:
So, the weird thing about the Pterodactyl project is it _copies_ the server app into the project folder. So each time you build the server it gets re-signed. I'd love to find a way to split the things out into separate chunks and have a real build process, but an iOS library _can't_ depend on a Mac binary.